api/v1/runtime/runtime_agent_api.proto

```diff
@@ -165,6 +165,7 @@ message CastaiConfig {
 
 message EnricherConfig {
   bool enable_file_hash_enricher = 1;
+  optional string sensitive_values_regex = 2;
```

Comment:
IMO optional is not needed here because empty string value is the default for flag anyway.

Reply:
done.